app/views/settings/profile/edit.haml

```diff
@@ -0,0 +1,55 @@
+.card
+  .card-body
+    = bootstrap_form_for(current_user, url: settings_profile_picture_path, html: { multipart: true }, method: :patch) do |f|
+
+      .media#profile-picture-media
+        .pull-left
+          %img.avatar-lg.mr-3{ src: current_user.profile_picture.url(:medium) }
+        .media-body
+          = f.file_field :profile_picture, accept: APP_CONFIG[:accepted_image_formats].join(",")
+
+      .row.d-none#profile-picture-crop-controls
+        .col-sm-10.col-md-8
+          %strong= t(".adjust.profile_picture")
+          %img#profile-picture-cropper{ src: current_user.profile_picture.url(:medium) }
+
+      .row.mb-2#profile-header-media
+        .col
+          %img.mw-100.mr-3{ src: current_user.profile_header.url(:mobile) }
+        .col-xs-12.mt-3.mt-sm-0.pl-3.pr-3
+          = f.file_field :profile_header, accept: APP_CONFIG[:accepted_image_formats].join(",")
+
+      .row.d-none#profile-header-crop-controls
+        .col-sm-10.col-md-8
+          %strong= t(".adjust.profile_header")
+          %img#profile-header-cropper{ src: current_user.profile_header.url(:web) }
+
+      = f.check_box :show_foreign_themes
```

Comment:
not part of this PR, but... why is this here with the pictures?

Reply:
Always has been, no idea why it is there really